items/BOSS_SPIRIT_BOW.json

```diff
@@ -1,7 +1,7 @@
 {
   "itemid": "minecraft:bow",
-  "displayname": "§f§f§5Spirit Bow",
```

Comment:
This change is not needed as Hypixel sets this random reset char. If you change something next time you dont need to remove it